### PR TITLE
feat: Added Wishlist Page

### PR DESCRIPTION
**Pull request by vishal804**
Hello,

Develop Wishlist page for Ecommerce project. 

Please Review 

Thank You

### CODE REVIEW

File: pages/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+              <a href="#">Login</a>
+            </button>
+          </li>
+          <li class="NoShow">
+            <a href="#"><i class="fa-2x far fa-heart"></i></a>
+          </li>
+          <li class="notification NoShow">
+            <a href="#"
+              ><i class="fas fa-shopping-cart fa-2x"></i
+              ><span class="badge">0</span></a
```
Comment (ApurvaSawant11) [inline pages/wishlist.html:41]: The badge is lower than expected. Adjust the `top` property

**Review by ApurvaSawant11 — APPROVED** [review]
Everything looks good!

**Review by foolhardy21 — APPROVED** [review]
looks good. you can replace card-container1 with something more meaningful like (card-text-container, card-btn-container, card-img-container) .

File: pages/wishlist.html (hunk 1 above)
Comment (vishal804) [reply pages/wishlist.html:41]: Thanks For Pointing It  Done the Changes

**Comment by vishal804** [comment]
> looks good. you can replace card-container1 with something more meaningful like (card-text-container, card-btn-container, card-img-container) .

Will do the Changes 
Thanks for Review

**Review by Developers-codz — APPROVED** [review]
Looking good to me ! You can merge it

**Comment by vishal804** [comment]
> Looking good to me ! You can merge it

Thanks For Review

**Pull request merged**